python/paddle/v2/framework/layers.py

```diff
@@ -276,6 +276,30 @@ def pool2d(input,
     return pool_out
 
 
+def sequence_conv_pool(input,
```

Comment:
```python
_create_op_func_('sequence_conv_pool')
```

Reply:
Here we need to check validation of input arguments and need to create filter parameter based on two arguments.